packages/shared/src/graphql/actions.ts

```diff
@@ -23,6 +23,7 @@ export enum ActionType {
   ExistingUserSeenStreaks = 'existing_user_seen_streaks',
   BookmarkPromoteMobile = 'bookmark_promote_mobile',
   HidePublicSquadEligibilityCard = 'hide_public_squad_eligibility_card',
+  CustomFeedBeta = 'custom_feed_beta',
```

Comment:
Why do we add beta to the action types?
Isn't it just you completed `custom_feed` (open for discussion though)